### PR TITLE
Changing file names for consistency between load and metabolism datafiles

### PR DESCRIPTION
**Pull request by jenklug**
A few lakes were named slightly differently in the metabolism results files vs the nutrient loads files (e.g., Lilli vs Lillinonah) which made data management more difficult.  I changed the names of Lilli to Lillinonah, Stuptjarn to Struptjarn, and Langtjern2013 to Langtjern 2013, and Langtjern2014 to Langtjern 2014 in the nutrient load results folder.  Also, I moved the fingerprintplots r code into the R_code folder

### CODE REVIEW

File: R_code/fingerprintplots.r [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+#starts with reading in individual lake files and binding them into one data frame
+#then merges dataframes together (e.g., metabolism dataframe with loading data frame)
+#there is code in the dropbox for creating a dataframe for the light climate data but I've left this
+#out for now because the light climate data are not on Github
+#code for plotting starts~line 100
+
+
+library(xlsx)
+
+dir<-'C:/Users/jklug/Documents/catchment_metab_wg/results/metab/20161107/' # directory of metabolism data
```
Comment (jzwart) [inline R_code/fingerprintplots.r]: If you are using projects in R Studio, you should need your full directory path, just the `results/metab/20161107`. This allows for others to use this code without having to change the directory path. 
```suggestion
dir<-'results/metab/20161107/' # directory of metabolism data
```

File: R_code/fingerprintplots.r [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+# Spring = (doy < 180)
+# Summer = (doy >= 180 & doy <=240)
+# Fall = (doy > 240)
+all_metab$season <- rep(NA,nrow(all_metab))
+all_metab$season <- ifelse(all_metab$doy>120&all_metab$doy<180,'spring',all_metab$season) # if doy with criteria, spring, else leave alone
+all_metab$season <- ifelse(all_metab$doy>=180&all_metab$doy<=240,'summer',all_metab$season)
+all_metab$season <- ifelse(all_metab$doy>240&all_metab$doy<305,'fall',all_metab$season)
+
+#loop below creates one dataframe with all loading data in it - JK modifying the loop JZ created above for metabolism
+dir<-'C:/Users/jklug/Documents/catchment_metab_wg/results/nutrient load/' # directory of loading data
```
Comment (jzwart) [inline R_code/fingerprintplots.r]: again, don't need full file path 
```suggestion
dir<-'results/nutrient load/' # directory of loading data
```

File: R_code/fingerprintplots.r [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+#merging loads and metabolism data into one dataframe
+#this is currently not working 100% because of difference in lake names in the two dataframes
+
+all_metabload<-merge(all_metab, all_load, by=c('lake','doy')) #does not include lakes without inflows or Mendota (no load)
+all_metabload$DOC_TP<-all_metabload$DOC_load/all_metabload$TP_load #creating DOC:TP column
+all_metabload$TN_TP<-all_metabload$TN_load/all_metabload$TP_load #creating TP:TN column
+
+
+#loop below creates one dataframe with all schmidt stability data in it - JK modifying JZ allmetab loop
+dir<-'C:/Users/jklug/Documents/catchment_metab_wg/data/schmidt stability/' # directory of stability data
```
Comment (jzwart) [inline R_code/fingerprintplots.r]: shorten file path 
```suggestion
dir<-'data/schmidt stability/' # directory of stability data
```

**Comment by jzwart** [comment]
To make the changes, just change the file on your local computer, commit, and push (you don't need a new PR). The committed changes will be automatically integrated into this PR.

**Comment by jenklug** [comment]
OK thank you.  I made the changes, commited and pushed.

**Comment by jzwart** [comment]
looks great!

**Pull request merged**